plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/SQLGroupingQueryGenerator.java

```diff
@@ -171,9 +171,11 @@ public String generateGroupingQuery(String queryText) throws DBException {
             }
             sql.append(groupAttributes.get(i).prepareSqlString(subqueryAlias));
         }
-        boolean isDefaultGrouping = groupFunctions.size() == 1 && groupFunctions.get(0).equalsIgnoreCase(DEFAULT_FUNCTION);
+        boolean isCountPresent = groupFunctions
+            .stream()
+            .anyMatch(f -> f.equalsIgnoreCase(DEFAULT_FUNCTION));
```

Comment:
Just a nitpick, but we put `.stream` on the same line as the collection expression


```suggestion
        boolean isCountPresent = groupFunctions.stream()
            .anyMatch(f -> f.equalsIgnoreCase(DEFAULT_FUNCTION));
```

Reply:
Applies to other places as well